pkg/flink/internal/controller/.snapshots/TestStatementControllerTestSuite-TestExecuteStatementWaitsForCompletedState

```diff
@@ -0,0 +1,6 @@
+Statement successfully submitted.
+Waiting for statement to be ready. Statement phase is PENDING.
+Statement phase is RUNNING.
+Listening for execution errors. Press Enter to detach.
```

Comment:
Should we update this to `Getting your statement ready. You can press enter to detach.`?

Reply:
I would rather leave it as is for now, and then we can adjust it later wenn we might have to support this flow for select statements